server/models/User.js

```diff
@@ -28,8 +28,14 @@ const User = sequelize.define(
     slackId: { type: DataTypes.STRING, allowNull: true, unique: true },
     slackData: DataTypes.JSONB,
     jwtSecret: encryptedFields.vault('jwtSecret'),
+    suspendedAt: DataTypes.DATE,
   },
   {
+    getterMethods: {
```

Comment:
Could we define this on the prototype?

Reply:
Afaik I can't define a getter (not a function) on prototype unless I use `Object.defineProperty`